docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,272 @@
+| **Klusterlet**       | **Klusterlet** |
+
+## Prerequisites
+
+Before registering a WEC, ensure you have:
+
+1. **A running Kubernetes cluster** that will serve as the WEC
+2. **Network connectivity** from the WEC to the KubeFlex hosting cluster (HTTPS connections must be possible)
+3. **`kubectl` access** to both the WEC and ITS
+4. **`clusteradm` CLI tool** installed on the machine where you'll run the registration commands ([installation guide](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/))
+5. **An existing ITS** with OCM cluster manager running
+6. **Sufficient permissions** to create resources in both clusters (typically cluster-admin or equivalent)
+
+To verify your ITS is ready for WEC registration:
+
+```shell
+# Check if the ITS is accessible (this returns a short result)
+kubectl --context <its-context> get ServiceAccount default
+
+# Verify OCM cluster manager is running (look for absence of Pending or failing Pods)
+kubectl --context <its-context> get pods -n open-cluster-management-hub
+
+# Check for the customization-properties namespace
+kubectl --context <its-context> get ns customization-properties
+```
+
+## Registration Process
+
+### Step 1: Get the Registration Token
+
+Obtain a registration token from the ITS:
+
+```shell
+# Get the registration token from the ITS
+clusteradm --context <its-context> get token
+```
+
+This command outputs a `clusteradm join` command that you'll use in the next step.
+
+### Step 2: Join the WEC to the ITS
+
+Execute the join command on the WEC to initiate the registration process:
+
+```shell
+# Basic join command (additional flags may be required based on your environment)
+clusteradm join --hub-token <token> --hub-apiserver <api-server-url> --cluster-name <your-wec-name> --context <wec-context>
+```
+
+**Important flags:**
+- `--cluster-name`: Choose a unique name for your WEC
+- `--context`: Specify the kubectl context for your WEC
+- `--singleton`: Use this flag if the WEC is a single-node cluster
+- `--force-internal-endpoint-lookup`: Required for Kind clusters and other local development setups
+
+**Example for Kind clusters:**
+```shell
+clusteradm join --hub-token <token> --hub-apiserver <api-server-url> --cluster-name cluster1 --context cluster1 --singleton --force-internal-endpoint-lookup
+```
+
+**Example for cloud provider clusters:**
+```shell
+clusteradm join --hub-token <token> --hub-apiserver <api-server-url> --cluster-name prod-cluster-east --context prod-cluster-east
+```
+
+### Step 3: Wait for Certificate Signing Request
+
+After the join command completes, wait for a Certificate Signing Request (CSR) to appear on the ITS:
+
+```shell
+# Check for pending certificate signing requests
+kubectl --context <its-context> get csr
+```
+
+You should see a CSR with your WEC name and status `Pending`. This CSR is created by the OCM registration agent running on your WEC.
```

Comment:
No, "You should see ..." is not necessarily true. The first few `kubectl --context <its-context> get csr` might NOT show the CSR with your WEC name and status `Pending`. You need to explain to this to the reader and explicitly tell them to repeat this `get` until the needed CSR appears.